cns/aks.Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Get a trusted CA cert bundle
+FROM alpine:latest as certs
+RUN apk --update add ca-certificates
+
+# Build cns
+FROM golang:1.14-alpine AS builder
+# Build ars
+ARG VERSION
+ARG CNS_AI_PATH
```

Comment:
maybe default here as well

Reply:
I'm going to leave it mostly because checks take so long to run